api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -46,6 +51,8 @@
             new Function<Criteria, org.pmiops.workbench.model.Criteria>() {
                 @Override
                 public org.pmiops.workbench.model.Criteria apply(Criteria criteria) {
+                    Type listType = new TypeToken<ArrayList<Attribute>>(){}.getType();
```

Comment:
Are we going to need to be able to query on these attributes? 

Representing them as JSON in a string (instead of child rows) will make that difficult. If this is purely extra information returned to the client, then this may be fine.